pkg/server/service.go

```diff
@@ -134,12 +134,14 @@ func NewCRIContainerdService(config options.Config) (CRIContainerdService, error
 		client:             client,
 	}
 
-	imageFSPath := imageFSPath(config.ContainerdConfig.RootDir, config.ContainerdConfig.Snapshotter)
-	c.imageFSUUID, err = c.getDeviceUUID(imageFSPath)
-	if err != nil {
-		return nil, fmt.Errorf("failed to get imagefs uuid of %q: %v", imageFSPath, err)
+	if !c.config.SkipImageFSUUID {
```

Comment:
suggest adding an info logf that it has been skipped..

Reply:
Will do.

Reply:
Done.